<a href="https://colab.research.google.com/github/rishika-p/assignment-1/blob/master/q2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
dist=int(input("enter distance to be travel:"))
wt=int(input("enter weight of goods:"))
if dist>=500:
  if wt>=100:
    cost=5*dist
  elif wt>=10 and wt<100:
    cost=6*dist
  else:
    cost=7*dist
else:
  if wt>=100:
    cost=8*dist
  else:
    cost=5*dist
print("Amount to be charged:",cost)


enter distance to be travel:520
enter weight of goods:50
Amount to be charged: 3120


A quick fox transport co. wants to develop an application for calculating amount based on distance and weights of goods.The charges to be calculated as per rates.
i/p:distance to travelled:520
weight of goods:50
o/p:amount to be charged :3120